ampersand-state.js

```diff
@@ -587,7 +609,7 @@ var dataTypes = {
     date: {
         set: function (newVal) {
             var newType;
-            if (!_.isDate(newVal)) {
+            if (!isDate(newVal)) {
                 try {
                     var dateVal = new Date(newVal).valueOf();
                     if (isNaN(dateVal)) {
```

Comment:
For reasons described here: https://gist.github.com/kitcambridge/1086528 we should also switch this out to instead use `amp-is-nan`: http://amp.ampersandjs.com/#amp-is-nan